app/components/Map.js

```diff
@@ -181,7 +181,7 @@ export default class Map extends Component {
         <MapView
           ref="map"
           showsUserLocation={true}
-          initialRegion={{ longitudeDelta: 0.005, latitude: currLoc.latitude,longitude: currLoc.longitude, latitudeDelta: 0.005 }}
+-         initialRegion={{ longitudeDelta: 0.005, latitude: currLoc.latitude,longitude: currLoc.longitude, latitudeDelta: 0.005 }}
```

Comment:
@tiffeli Just to clarify, should I take out just the errant `-` or the whole `initialRegion` bit?